macros/miscs/debugging.cfg

```diff
@@ -35,3 +35,19 @@ gcode:
             {% endfor %}
         {% endif %} 
     {% endfor %}
+
+# Specific macro to raise an error during the print_start and stop in case of ERCF error
+[gcode_macro _DO_ERROR_ERCF_PAUSE]

```

Comment:
Can this macro be renamed in: _ERCF_ERROR_CHECK or something like that? It would help understand the problem and feature adressed by this custom macro